src/common/helsinkiLogo/HelsinkiLogo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+import styles from './HelsinkiLogo.module.css';
+
+type Props = {
+  className?: string;
+};
+function HelsinkiLogo({ className }: Props) {
+  const { i18n } = useTranslation();
+
+  if (i18n.languages[0] === 'sv')
+    return (
+      <span
+        className={classNames(styles.logoSv, className)}
+        aria-label="Helsinki logo"
```

Comment:
Consider translating these as well?

Reply:
There is a task coming to update various text's. I'll add translations with that one.